assets/age-verifier.js

```
@@ -0,0 +1,71 @@
+  get cookieName() {
+    return 'concept:age-verifier';
+  }
+
+  init() {
+    if (this.testMode || !this.getCookie(this.cookieName)) {
+      this.load(this.delay);
+    }
+  }
+
+  load(delay) {
+    if (Shopify && Shopify.designMode) return;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)